README.md

```diff
@@ -86,7 +86,7 @@ If you are using EvalAI for hosting challenges, please cite the following techni
 }
 ```
 <p>
-    <a href="http://learningsys.org/sosp19/assets/papers/23_CameraReadySubmission_EvalAI_SOSP_2019%20(8)%20(1).pdf"><img src="docs/source/_static/img/evalai-paper.jpg"/></a>
+    <a href="https://namishkhanna.github.io/newtab/?url=http://learningsys.org/sosp19/assets/papers/23_CameraReadySubmission_EvalAI_SOSP_2019%20(8)%20(1).pdf"><img src="docs/source/_static/img/evalai-paper.jpg"/></a>
```

Comment:
Can we please use `target="_blank"` in the anchor tag :)

Reply:
@RishabhJain2018 Sorry, but we can't as in readme.md File Github doesn't have feature to open New Tab

Reply:
@RishabhJain2018 Please tell me what to do ?

Reply:
I see, then I would prefer to keep it as it is. Using `https://namishkhanna.github.io/newtab/?url` seems like a hack to me.

Thank you for the effort. Please feel free to take up any other issue.